src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -79,7 +91,7 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
   return (
     <ChatItemTargetText
       key={index}
-      AnonymousImage={handleImageUser()}
+      avatar={handleAvatar()}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from passing `AnonymousImage={handleImageUser()}` to `avatar={handleAvatar()}` in `ChatItemTargetText` is a good improvement. This makes the prop more generic and allows for different types of avatars (anonymous or signed) to be passed in.
```diff
- AnonymousImage={handleImageUser()}
+ avatar={handleAvatar()}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->